docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,581 @@
+.. _build_intel_boot_bin:
+
+Generating HDL boot image for Intel projects
+===============================================================================
+
+This page is dedicated to the building process of boot image for supported
+Intel/Altera projects. One of the key files that is specific to this
+projects is the ``bootloader``. It's role is to initialize the system after the
+BootROM runs. It bridges the gap between the limited functionality of the
+BootROM and the end user application, configuring the system as needed before
+the larger application can run
+
+
+Below there're useful links who explain in more detail the process and the
+necessary components:
+
+-  `Booting and Configuration <https://www.intel.com/content/dam/support/us/en/programmable/support-resources/bulk-container/pdfs/literature/hb/arria-v/av-5400a.pdf#:~:text=This%20topic%20describes%20the%20booting%20of>`__
+-  `SocBootFromFPGA <https://community.intel.com/t5/FPGA-Wiki/SocBootFromFPGA/ta-p/735773#:~:text=Introduction.%20The%20HPS%20(Hardened%20Processor%20System)>`__
+-  `Intel Bootloader <https://www.intel.com/content/www/us/en/support/programmable/support-resources/design-guidance/soc-bootloader.html#:~:text=Intel%C2%AE%20SoC%20FPGAs%20use%20a%20bootloader>`__
+
+Necessary files for booting up a HDL project
+-------------------------------------------------------------------------------
+
+.. warning::
+   
+   We don't provide any scripts to automate this whole process, so please read
+   carefully and follow all the steps in order to obtain the desired results.
+
+When booting a system with Linux + HDL (depending on the project), you will need
+to write the following files on the root of the BOOT FAT32 partition. After that,
+you must write the preloader image in a specific card partition, depending on
+the case. 
+
+-  For Arria10 SoC projects:
+
+   -  ``socfpga_arria10-common/zImage``
+   -  ``<target>/fit_spl_fpga.itb``
+   -  ``<target>/socfpga_arria10_socdk_sdmmc.dtb``
+   -  ``<target>/u-boot.img``
+   -  ``<target>/extlinux.conf → extlinux /extlinux.conf``
```

Comment:
```suggestion
   -  ``<target>/extlinux.conf → extlinux/extlinux.conf``
```
I'm not familiar with this, but would this arrow look/work ok in the PDF documentation?

Reply:
Yes, it renders properly in PDF (tested). The characters that would usually mess up the PDF document are highlighted in Visual Studio Code with a yellow box. As far as I know by working with this, only in these cases the chars don't render properly. So the arrows will be kept.